<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Eggs4</class>
 <widget class="QWidget" name="Eggs4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Gen 4 Eggs</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxProfile">
     <property name="title">
      <string>Profile</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="5">
       <widget class="QLabel" name="labelProfileGame">
        <property name="text">
         <string>Game</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="labelProfileSIDValue">
        <property name="text">
         <string notr="true">54321</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxProfiles"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonProfileManager">
        <property name="text">
         <string>Manager</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="labelProfileTIDValue">
        <property name="text">
         <string notr="true">12345</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" rowspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelProfile">
        <property name="text">
         <string>Profile</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelProfileSID">
        <property name="text">
         <string>SID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="labelProfileGameValue">
        <property name="text">
         <string>Diamond</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelProfileTID">
        <property name="text">
         <string>TID</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabEggSelection">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGenerator">
      <attribute name="title">
       <string>Generator</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxGeneratorRNGInfo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>RNG Info</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="1">
           <widget class="ComboBox" name="comboBoxGeneratorMethod">
            <item>
             <property name="text">
              <string>IVs</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>PID</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelGeneratorSeed">
            <property name="text">
             <string>Seed</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelGeneratorMaxAdvances">
            <property name="text">
             <string>Max Advances</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="TextBox" name="textBoxGeneratorSeed">
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="maxLength">
             <number>8</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelGeneratorInitialAdvances">
            <property name="text">
             <string>Initial Advances</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="TextBox" name="textBoxGeneratorInitialAdvances">
            <property name="text">
             <string notr="true">0</string>
            </property>
            <property name="maxLength">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="TextBox" name="textBoxGeneratorMaxAdvances">
            <property name="text">
             <string notr="true">100000</string>
            </property>
            <property name="maxLength">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QPushButton" name="pushButtonGenerate">
            <property name="text">
             <string>Generate</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelGeneratorMethod">
            <property name="text">
             <string>Method</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="checkBoxGeneratorMasuada">
            <property name="text">
             <string>Masuada</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="TableView" name="tableViewGenerator">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBoxGeneratorFilters">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>2</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Filters</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="3" column="0" colspan="2">
           <widget class="EggSettings" name="eggSettingsGenerator" native="true"/>
          </item>
          <item row="0" column="0" rowspan="3" colspan="2">
           <widget class="Filter" name="filterGenerator" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSearcher">
      <attribute name="title">
       <string>Searcher</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxSearcherRNGInfo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>RNG Info</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="3" column="3">
           <widget class="TextBox" name="textBoxSearcherMaxDelay">
            <property name="text">
             <string notr="true">2000</string>
            </property>
            <property name="maxLength">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="TextBox" name="textBoxSearcherMinDelay">
            <property name="text">
             <string notr="true">600</string>
            </property>
            <property name="maxLength">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="TextBox" name="textBoxSearcherPIDMaxAdvance">
            <property name="text">
             <string notr="true">30</string>
            </property>
            <property name="maxLength">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelSearcherDelay">
            <property name="text">
             <string>Delay</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="TextBox" name="textBoxSearcherPIDMinAdvance">
            <property name="text">
             <string notr="true">0</string>
            </property>
            <property name="maxLength">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="4">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pushButtonSearch">
              <property name="text">
               <string>Search</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonCancel">
              <property name="text">
               <string>Cancel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="2">
           <widget class="TextBox" name="textBoxSearcherIVMinAdvance">
            <property name="text">
             <string notr="true">0</string>
            </property>
            <property name="maxLength">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="TextBox" name="textBoxSearcherIVMaxAdvance">
            <property name="text">
             <string notr="true">100000</string>
            </property>
            <property name="maxLength">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2" colspan="2">
           <widget class="ComboBox" name="comboBoxSearcherMethod">
            <item>
             <property name="text">
              <string>IVs</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>PID</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Both</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelSearcherPIDState">
            <property name="text">
             <string>PID Advance</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelSearcherIVAdvance">
            <property name="text">
             <string>IV Advance</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelSearcherMethod">
            <property name="text">
             <string>Method</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="checkBoxSearcherMasuada">
            <property name="text">
             <string>Masuada</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBoxGeneratorFilters_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>2</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Filters</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="3" column="0" colspan="2">
           <widget class="EggSettings" name="eggSettingsSearcher" native="true"/>
          </item>
          <item row="0" column="0" rowspan="3" colspan="2">
           <widget class="Filter" name="filterSearcher" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="TableView" name="tableViewSearcher">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QProgressBar" name="progressBarSearcher">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextBox</class>
   <extends>QLineEdit</extends>
   <header>Forms/Controls/TextBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>Forms/Controls/TableView.hpp</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>Forms/Controls/ComboBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>Filter</class>
   <extends>QWidget</extends>
   <header>Forms/Controls/Filter.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EggSettings</class>
   <extends>QWidget</extends>
   <header>Forms/Controls/EggSettings.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxProfiles</tabstop>
  <tabstop>pushButtonProfileManager</tabstop>
  <tabstop>tabEggSelection</tabstop>
  <tabstop>comboBoxGeneratorMethod</tabstop>
  <tabstop>textBoxGeneratorSeed</tabstop>
  <tabstop>textBoxGeneratorInitialAdvances</tabstop>
  <tabstop>textBoxGeneratorMaxAdvances</tabstop>
  <tabstop>checkBoxGeneratorMasuada</tabstop>
  <tabstop>pushButtonGenerate</tabstop>
  <tabstop>tableViewGenerator</tabstop>
  <tabstop>comboBoxSearcherMethod</tabstop>
  <tabstop>textBoxSearcherPIDMinAdvance</tabstop>
  <tabstop>textBoxSearcherPIDMaxAdvance</tabstop>
  <tabstop>textBoxSearcherIVMinAdvance</tabstop>
  <tabstop>textBoxSearcherIVMaxAdvance</tabstop>
  <tabstop>textBoxSearcherMinDelay</tabstop>
  <tabstop>textBoxSearcherMaxDelay</tabstop>
  <tabstop>checkBoxSearcherMasuada</tabstop>
  <tabstop>pushButtonSearch</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>tableViewSearcher</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
